Common/Parsers/JSONParser.swift

```diff
@@ -45,7 +45,17 @@ extension Site: Encodable, Decodable {
         let encodedDeviceStatus: [[String : Any]] = deviceStatuses.flatMap{ $0.encode() }
         let encodedComplicationTimeline: [[String : Any]] = complicationTimeline.flatMap { $0.encode() }
         
-        return [JSONKey.url : url.absoluteString, JSONKey.overrideScreenLock : overrideScreenLock, JSONKey.disabled: disabled, JSONKey.uuid: uuid.uuidString, JSONKey.configuration: configuration?.encode() ?? "", JSONKey.sgvs: encodedSgvs, JSONKey.cals: encodedCals, JSONKey.mbgs: encodedMgbs, JSONKey.deviceStatuses: encodedDeviceStatus, JSONKey.complicationTimeline: encodedComplicationTimeline]
+        return [JSONKey.url : url.absoluteString,
+                "nextRefreshDate" : nextRefreshDate,
```

Comment:
Sure, though I feel like these aren't really JSON keys at this point. It's somewhat misleading.